utils/quick-setup.sh

```diff
@@ -73,33 +76,136 @@ function install-docker-ubuntu {
     sudo apt-get -y install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
 }
 
+function install-docker-rhel {
+    # using instructions from:
+    # https://docs.docker.com/engine/install/rhel/#install-using-the-repository
+    sudo yum remove -y docker \
+                  docker-client \
+                  docker-client-latest \
+                  docker-common \
+                  docker-latest \
+                  docker-latest-logrotate \
+                  docker-logrotate \
+                  docker-engine \
+                  podman \
+                  runc
+
+    sudo yum install -y yum-utils
+    sudo yum-config-manager -y --add-repo https://download.docker.com/linux/rhel/docker-ce.repo
+
+    sudo yum install -y docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+
+    # diverges from the instructions. This means docker daemon starts on each boot.
+    sudo systemctl enable --now docker
+}
+
+function install-docker-fedora {
+    # using instructions from:
+    # https://docs.docker.com/engine/install/rhel/#install-using-the-repository
+    sudo dnf remove -y docker \
+                  docker-client \
+                  docker-client-latest \
+                  docker-common \
+                  docker-latest \
+                  docker-latest-logrotate \
+                  docker-logrotate \
+                  docker-selinux \
+                  docker-engine-selinux \
+                  docker-engine
+
+    sudo dnf install -y dnf-plugins-core
+    sudo dnf config-manager -y --add-repo https://download.docker.com/linux/fedora/docker-ce.repo
+
+    sudo dnf install -y docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+
+    # diverges from the instructions. This means docker daemon starts on each boot.
+    sudo systemctl enable --now docker
+}
+
+function post-install-docker {
+    # instructions from:
+    # https://docs.docker.com/engine/install/linux-postinstall/
+    sudo groupadd docker
+    sudo usermod -aG docker "$SUDO_USER"
```

Comment:
I think it is useful to add this

```
newgrp docker
```

I once had a problem with login/logout not helping the case. The docs say that sometimes a reboot is needed, but also mention that this command might help

Reply:
I tried this in the shell script along with `su`. It won't/doesn't work due to the script executing as root.

Reply:
We could add to the docs that the user can either log out & back in _or_ execute `newgrp docker`?

Reply:
yes, lets put in a note then

Reply:
Done.